<?xml version="1.0"?>
<launch>

    <arg name="ID" default="1"/>

    <node pkg="sensor_fusion" type="noise.py" name="targetwithNoise_$(arg ID)" required="True" output="screen">
        <param name = "uav_id" value = "$(arg ID)"/>
	</node>

  
    <node pkg="sensor_fusion" type="kalman_filter.py" name="kalman_filter_$(arg ID)" required="True" args="total_uav" output="screen">
        <param name = "uav_id" value = "$(arg ID)"/>
    </node>


</launch>
    
   